content/en/real_user_monitoring/error_tracking/_index.md

```diff
@@ -1,32 +1,53 @@
+## Overview
+
+Datadog collects a lot of errors, and it is critical for your system's health to consistently monitor these errors. When there are so many individual error events, it becomes hard to identify which ones matter the most and which ones should be fixed first. By tracking, triaging, and debugging crashes, you can minimize the impact of fatal errors on your web and mobile application's user experience.
+
+{{< img src="real_user_monitoring/error_tracking/page.png" alt="The Error Tracking Explorer for RUM displaying issues from your web and mobile application's crash reports" style="width:100%;" >}}
+
+## Upload crash reports
 
-## What is error tracking?
+{{< whatsnext desc="To get started with Datadog Error Tracking for RUM, see the corresponding documentation to upload source maps for your framework:" >}}
+    {{< nextlink href="real_user_monitoring/error_tracking/browser" >}}Browser{{< /nextlink >}}
+    {{< nextlink href="real_user_monitoring/error_tracking/android" >}}Android{{< /nextlink >}}
+    {{< nextlink href="real_user_monitoring/error_tracking/ios" >}}iOS{{< /nextlink >}}
+    {{< nextlink href="real_user_monitoring/error_tracking/expo" >}}Expo{{< /nextlink >}}
+    {{< nextlink href="real_user_monitoring/error_tracking/reactnative" >}}React Native{{< /nextlink >}}
+    {{< nextlink href="real_user_monitoring/error_tracking/flutter" >}}Flutter{{< /nextlink >}}
+{{< /whatsnext >}}
 
-Datadog collects a lot of errors. It's critical to your system's health to monitor these errors, but there can be so many individual error events that it’s hard to identify which ones matter the most and should be fixed first. Error Tracking makes it easier to monitor these errors by:
+## Examine issues to start troubleshooting or debugging
 
-- __Grouping similar errors into issues__ to reduce the noise and help identify the most important ones.
-- __Following issues over time__ to know when they first started, if they are still ongoing, and how often they are occurring.
-- __Getting all the context needed in one place__ to facilitate troubleshooting the issue.
+Error Tracking automatically categorizes crashes collected from your web and mobile applications into issues in the [Error Tracking Explorer][1]. 
 
+Click on an issue to view debugging information such as the stack trace or a user session timeline, and metadata such as a user location, version, and any custom attributes you included in your crash reports. 
+
+You can set alerts on Error Tracking events to stay on top of fatal issues that may occur, along with the following actions:
+
+- **Grouping similar errors into issues** to reduce the noise and identifying the most important ones.
+- **Following issues over time** to know when they first started, if they are still ongoing, and how often they are occurring, which helps you identify the most important ones.
+- **Getting all the context needed in one place** to facilitate troubleshooting.
```

Comment:
```suggestion
Error tracking enables you to:

- Set alerts on Error Tracking events. This helps you to remain informed of fatal issues that may occur.
- Group similar errors into issues, so that you can more easily identify important errors and reduce noise.
- Follow issues over time to know when they first started, if they are still ongoing, and how often they are occurring.
- Collect all the necessary context in one place to facilitate troubleshooting.
```